pages/index.html

```diff
@@ -1,12 +1,28 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
-</head>
-<body>
-  
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link href="../styles/style.css" rel="stylesheet">
+  </head>
+  <body>
+    <div class="wrapper">
+      <div class="design">
+        <aside class="aside">
+          <nav><a href="index.html">Home</a><a href="portfolio.html">Portfolio</a><a href="about.html">About</a></nav>
+        </aside>
+        <header class="header">
+          <h1>Welcome!</h1>
+        </header>
+      </div>
+      <article class="main">
+        <h2>Hi. I am Doina.</h2>
+        <h3>I'M A STUDENT AT ADA DEVELOPERS ACADEMY. FUTURE FULL STACK DEVELOPER</h3>
```

Comment:
In semantic HTML, header tags should be used to indicate different sections of content, not as stylistic elements.  [Link](https://bighack.org/how-to-heading-structure-accessibility/)